tests/modules/test_001_baseline/baseline_values.json

```diff
@@ -5617,7 +5617,7 @@
             "provider_name": "registry.terraform.io/hashicorp/time",
             "schema_version": 0,
             "values": {
-              "create_duration": "30s",
+              "create_duration": "0s",
```

Comment:
Just for note, my plan was for these code updates to not introduce any unexpected changes to the `baseline_values.json` files but these `create_duration` value changes appear to actually be a previously undiscovered bug which has been fixed by this update.

This was caused by the old logic resulting in any additions being specified for `create_duration` resulting in all others to default to a generic `30s` if not specified. The new logic allows any combination of inputs to be specified whilst retaining the original defaults for all others.